runtime/testy/Ansible.py

```diff
@@ -213,6 +214,7 @@ def unpackage(data):
             new_state = received_proto.student_code_status
             unpackaged_data["student_code_status"] = [new_state, time.time()]
             if self.control_state is None or new_state != self.control_state:
+                print(new_state, "UDP")
```

Comment:
We should delete this print statement, it was just used for print line debugging.